CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 1.1.1 (YYYY-MM-DD)
+
+### Breaking Changes
+* None.
+
+### Enhancements
+* None.
+
+### Fixed
+* Ordering not respected for `RealmQuery.first()`. (Issue [#953](https://github.com/realm/realm-kotlin/issues/953))
```

Comment:
I guess we don't really know this until https://github.com/realm/realm-kotlin/pull/956 has been merged?

Reply:
No, I guess you are right. I just cherry-picked this core-update from #956, but technically you are right. Will remove the fixed-entry.